generators/client/templates/angular/src/main/webapp/app/shared/shared-libs.module.ts.ejs

```diff
@@ -31,6 +31,7 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
         NgJhipsterModule.forRoot({
             // set below to true to make alerts look like toast
             alertAsToast: false,
+            alertTimeout: 5,
```

Comment:
Want this in milliseconds? Sorry I'm on my fone. If it indeed is millis then 5 is too low

Reply:
oh indeed, I think it's millis ! So it should be 5000